server/channels/api4/emoji.go

```diff
@@ -221,6 +223,11 @@ func getEmojiByName(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if !*c.App.Config().ServiceSettings.EnableCustomEmoji {
```

Comment:
Strictly speaking, this is a change, but this API should have checked this setting before